test/11.js

```diff
@@ -0,0 +1,12 @@
+const div7= require('../solutions/11.js');
+
+const tester = (num,result)=>{
+  if(div7(num)==result){
+    console.log('correct the answer is '+ div7(num));
```

Comment:
It will look better with back ticks: `console.log(`correct answer is ${div7(num)}`)`